code-review/README.md

```diff
@@ -43,3 +50,30 @@
   - Do unit tests actually test that the code is performing the intended functionality?
   - Are arrays checked for ‘out-of-bounds’ errors?
   - Could any test code be replaced with the use of an existing API?
+
+
+## Pull Request Format
+  - Prefix the branch & PR with the Jira ticket ID eg. `[TD-000] Brief summary of branch`
+  - Include Jira ticket number, summary description of the changes, testing notes & example screenshots for UI changes.
+
+### Example PRs
+  - https://github.com/factorymedia/coresites/pull/287
+  - https://github.com/factorymedia/coresites/pull/286
+  - https://github.com/factorymedia/coresites/pull/282
+
+### Pull Request Template
+```md
+**Jira Ticket**
+https://factorymedia.atlassian.net/browse/TD-xxx
+
+**Summary**
+Quick description of changes and expected behaviour.
+
+**Example**
+A few links or images showing before / after changes.
+
+**Testing / QA Notes**
+Brief notes to save your tester some time, and let them know / what to test for.
+
+```
```

Comment:
can we also add review note? i.e.)
** Review note
I want a review only for coding style / typo please (30% review)
I want a review for all (styles / typo / logic / etc) (90% review)